### PR TITLE
Feature/how it works

### PR DESCRIPTION
**Pull request by Daniel-Gomez-jpg**
how it works page created

### CODE REVIEW

File: src/styles/style.scss [hunk 1]
```diff
@@ -225,7 +225,349 @@
...
+            font-style: normal;
+            font-weight: 700;
+            font-size: 1.25rem;
+            line-height: 1.5625rem;
+            letter-spacing: -0.0313rem;
+            color: $color-dark-green;
+            max-width: 33.8125rem;
+
+            @media screen and (min-width:1024px) {
+            font-weight: 700;
```
Comment (HackMort) [inline src/styles/style.scss]: This value is the same as on "mobile" we don't need to add this or fix the value if needed.

File: src/styles/style.scss [hunk 2]
```diff
@@ -225,7 +225,349 @@
...
+            font-size: 30px;
```
Comment (HackMort) [inline src/styles/style.scss]: Use Rems not PX

File: src/styles/style.scss [hunk 3]
```diff
@@ -225,7 +225,349 @@
...
+            font-weight: 500;
+            font-size: 1.125rem;
+            line-height: 1.375rem;
+            color: $color-dark-gray;
+            margin-top: 1.25rem;
+            margin-bottom: .6875rem;
+            max-width: 39.6875rem;
+            
+            @media screen and (min-width:1024px) {
+            font-weight: 500;
```
Comment (HackMort) [inline src/styles/style.scss]: Same comment as above. If the font-weight has the same value, we don't need to add it again. It seems to me that you are just copying and pasting from the Figma.

File: src/styles/style.scss [hunk 4]
```diff
@@ -225,7 +225,349 @@
...
+            line-height: 1.125rem;
+            margin-bottom: .9375rem;
+            max-width: 39.6875rem;
+        }
+    
+        &__item{
+            font-style: normal;
+            font-weight: 300;
+            font-size: .875rem;
+            line-height: 125%;
```
Comment (HackMort) [inline src/styles/style.scss:317]: Rems not %

File: src/styles/style.scss [hunk 5]
```diff
@@ -225,7 +225,349 @@
...
+
+            @media screen and (min-width:768px) {
+            font-size: 1.125rem;
+            }
+        }
+        &__item-nosub{
+            font-style: normal;
+            font-weight: 300;
+            font-size: .875rem;
+            line-height: 125%;
```
Comment (HackMort) [inline src/styles/style.scss:331]: Rems not %

File: src/styles/style.scss [hunk 6]
```diff
@@ -225,7 +225,349 @@
...
+
+        &__container{
+            margin: auto;
+            @media screen and (min-width:1024px) {
+                display: flex;
+                justify-content: space-between;
+                
+                :first-child{
+                    width: 27.0625rem;
+                    margin-right: 5px;
```
Comment (HackMort) [inline src/styles/style.scss]: Rems not PX

File: src/styles/style.scss [hunk 7]
```diff
@@ -225,7 +225,349 @@
...
+                    margin-top: 3px;
```
Comment (HackMort) [inline src/styles/style.scss]: Rems not PX

File: src/styles/style.scss [hunk 8]
```diff
@@ -225,7 +225,349 @@
...
+        position: relative;
+        z-index:2;
+        font-style: normal;
+        font-weight: 300;
+        font-size: .875rem;
+        line-height: 1.125rem;
+        color: $color-dark-gray;
+        
+        @media screen and (min-width:900px) {
+        font-size: 18px;
```
Comment (HackMort) [inline src/styles/style.scss:527]: Rems not PX

File: src/styles/style.scss [hunk 9]
```diff
@@ -225,7 +225,349 @@
...
+        position: relative;
+        z-index:2;
+        font-style: normal;
+        font-weight: 700;
+        font-size: .875rem;
+        line-height: 1.125rem;
+        color: $color-dark-green;
+
+        @media screen and (min-width:900px) {
+            font-size: 18px;
```
Comment (HackMort) [inline src/styles/style.scss]: Rems not PX

File: src/pages/how-it-works.astro [hunk 10]
```diff
@@ -4,25 +4,141 @@ import Layout from "../layouts/Layout.astro";
...
+         <h1 class="hpp__title">How low alkaline phosphatase (ALP) becomes hazardous</h1>
+
+          <h2 class="hpp__sub-title">Under normal circumstances:</h2>
+          
+           <div class="hpp__container">
+            
+            <div class="hpp__block">
+          <ul class="hpp__list">
+            <li class="hpp__item">Alkaline phosphatase dephosphorylates inorganic pyrophosphate (PPi) to releaseinorganic phosphate (Pi)</li>
+              <li class="hpp__sub-item">Pi then combines with calcium to form hydroxyapatite crystal, the foundation of
```
Comment (HackMort) [inline src/pages/how-it-works.astro]: This bullets should be circle not green dot, see screenshot:
image

File: src/pages/how-it-works.astro [hunk 11]
```diff
@@ -4,25 +4,141 @@ import Layout from "../layouts/Layout.astro";
...
+          
+           <div class="hpp__container">
+            
+            <div class="hpp__block">
+          <ul class="hpp__list">
+            <li class="hpp__item">Alkaline phosphatase dephosphorylates inorganic pyrophosphate (PPi) to releaseinorganic phosphate (Pi)</li>
+              <li class="hpp__sub-item">Pi then combines with calcium to form hydroxyapatite crystal, the foundation of 
+                bone formation</li>
+              </ul>
+                    <img src="/assets/images/first-hpp.svg" alt="" class="hpp__image">
```
Comment (HackMort) [inline src/pages/how-it-works.astro]: This image should go full-width on mobile, without any space  around it.
image

File: src/pages/how-it-works.astro [hunk 12]
```diff
@@ -4,25 +4,141 @@ import Layout from "../layouts/Layout.astro";
...
+                  </div>
+                    
+                    <div class="hpp__block">
+                      
+                      <ul class="hpp__list">
+                        <li class="hpp__item">Alkaline phosphatase also dephosphorylates pyridoxal 5’-phosphate (PLP, the active form of vitamin B6) into pyridoxal, allowing it to cross into the central nervous system (CNS)</li>
+                        <li class="hpp__sub-item">PLP is required for the metabolism of neurotransmitters</li>
+                      </ul>
+                      
+                      <img src="/assets/images/hpp-second.svg" alt="" class="hpp__image">
```
Comment (HackMort) [inline src/pages/how-it-works.astro]: Image should be full-width:
image

File: src/pages/how-it-works.astro [hunk 13]
```diff
@@ -4,25 +4,141 @@ import Layout from "../layouts/Layout.astro";
...
+              </ul>
+                    <img src="/assets/images/first-hpp.svg" alt="" class="hpp__image">
+                    <p class="hpp__image-disclaimer">Image created for Alexion Pharmaceuticals, Inc. for illustrative purposes.</p>
+                  </div>
+                    
+                    <div class="hpp__block">
+                      
+                      <ul class="hpp__list">
+                        <li class="hpp__item">Alkaline phosphatase also dephosphorylates pyridoxal 5’-phosphate (PLP, the active form of vitamin B6) into pyridoxal, allowing it to cross into the central nervous system (CNS)</li>
+                        <li class="hpp__sub-item">PLP is required for the metabolism of neurotransmitters</li>
```
Comment (HackMort) [inline src/pages/how-it-works.astro]: Fix bullet:

image

File: src/pages/how-it-works.astro [hunk 14]
```diff
@@ -4,25 +4,141 @@ import Layout from "../layouts/Layout.astro";
...
+                      </ul>
+                      
+                      <img src="/assets/images/hpp-second.svg" alt="" class="hpp__image">
+                    </div>
+                  </div> 
+                </div>
+                    <h2 class="hpp__sub-title">In patients with hypophosphatasia, a mutation on the ALPL gene causes deficient alkaline phosphatase, resulting in low alkaline phosphatase activity.</h2>
```
Comment (HackMort) [inline src/pages/how-it-works.astro]: There's too much space between this elements:

image

File: src/pages/how-it-works.astro [hunk 15]
```diff
@@ -4,25 +4,141 @@ import Layout from "../layouts/Layout.astro";
...
+                  
+                          <img src="/assets/images/hpp-sisxth.svg" alt="" class="hpp__image">
+                       
+                </div>
+              </div>
+
+
+
+              <div class="without-strensiq">
+                <h1 class="without-strensiq__title">Without STRENSIQ</h1>
```
Comment (HackMort) [inline src/pages/how-it-works.astro]: Change this to an h2, also check the spacing elements are too close to each other.

image

**Review by HackMort — CHANGES_REQUESTED** [review]
Overall check spacing and make sure the images are placed correctly on all screen sizes.

**Review by HackMort — CHANGES_REQUESTED** [review]
Looks a lot better on mobile.
Check out spacing on destkop view, I still see some spacing issues between elements and images. (starting at 1024px)

image



Also the images are way too big starting at 1400px the space between the 2 columns is not matching. please fix.
image

**Review by HackMort — APPROVED** [review]

File: src/styles/settings/_variables.scss [hunk 16]
```diff
@@ -8,6 +8,7 @@ $color-dark-yellow: #DFDF00;
 $color-dark-gray: #626469;
 $color-bright-gray: #E2E7EA;
 $color-light-gray: #EEEEEE;
+$color-bright-gray: rgba(226, 231, 234, 0.5);
```
Comment (AndersonCastellon) [inline src/styles/settings/_variables.scss:11]: Duplicated variable
@Daniel-Gomez-jpg 
@HackMort

**Pull request merged**